.github/workflows/update-docs-webhook.yaml

```diff
@@ -1,6 +1,8 @@
 name: Update docs webhook
 on:
   push:
+    paths:
+      - 'docs/**'
```

Comment:
@ptgott do you think path filter here is a good idea? Or we are deploying on every push on purpose?

Reply:
A path filter here works for me! I can't think of a good reason to deploy the docs site on every push.